examples/flask/playback_runner.py

```diff
@@ -0,0 +1,53 @@
+from examples.flask.web_services import tape_recorder
+from playback.studio.equalizer import ComparatorResult, EqualityStatus
+from playback.studio.equalizer_tuning import EqualizerTuner, EqualizerTuning
+from playback.studio.studio import PlaybackStudio
+from playback.tape_recorder import TapeRecorder
+
+
+class ContentBasedEqualizerTune(EqualizerTuner):
+
+    @staticmethod
+    def playback_function(recording):
+        """
+        Rerun the operation given the recording
+        """
+        # The class it self is saved as metadata, we instantiate it and trigger the post method
+        operation_class = recording.get_metadata()[TapeRecorder.OPERATION_CLASS]()
```

Comment:
look at this now, this seems like something that can be abstracted away, no? isn't it always this? 
how come there is no Playback(recording).play that simply does get_metadata[...].[func_of_operation]?

Reply:
While writing this example I actually saw that I force operation decorator to be on a class and not on a function.
If i supported a function I could have done a simpler flask example using their
`@app.post` decorator on a function.
So I think that in the future we will support not just classes which means this is not always the case.
Also when you use inner classes pickle doesn't deserialilze it back to a class, it is remained as dictionary, so you won't always have this available.

That being said I agree we should add helper utilities but it is not really related to this PR.
Maybe you want to contribute code :)

Reply:
"Maybe you want to contribute code :)"